src/site/layouts/office.drupal.liquid

```diff
@@ -0,0 +1,197 @@
+                  <h3 class="asset-body-header">
+                    {{entity.title| truncate: 36}}</h3>
+                  <div class="asset-body-text">
+                    {{entity.fieldDescription | truncate: 81 }}</div>
+                  {% if entity.fieldBenefits != '' %}
+                  <div class="asset-category-text capitalize-first">
+                    <strong>Category:
+                    </strong>{{entity.fieldBenefits | breakTerms}}</div>
+                  {% endif %}
+                  <div class="asset-filetype-text"><strong>File type:
+                    </strong><span
+                      class="capitalize-first">{{entity.fieldMedia.entity.entityBundle}}</span>
+                  </div>
+
+                  <div class="asset-download-trigger">
+                    {% if entity.fieldMedia.entity.entityBundle == 'document' %}
+                    {% assign url = entity.fieldMedia.entity.fieldDocument.entity.url %}
+                    {% if entity.derivedFields.absoluteUrl %}
+                    {% assign url = entity.derivedFields.absoluteUrl %}
+                    {% endif %}
+                    <a class="file-download-with-icon" target="_blank"
+                      href="{{url}}" download="{{url}}"><i
+                        class="fas fa-download vads-u-padding-right--1"></i>Download
+                      {{entity.fieldMedia.entity.entityBundle | fileType}}</a>
+                    {% endif %}
+
+                    {% if entity.fieldMedia.entity.entityBundle == 'video' %}
+                    <a class="video-link" target="_blank"
+                      href="{{entity.fieldMedia.entity.fieldMediaVideoEmbedField}}"><i
+                        class="fab fa-youtube vads-u-padding-right--1"></i>Go
+                      to
+                      video</a>
+                    {% endif %}
+
+                    {% if entity.fieldMedia.entity.entityBundle == 'image' %}
+                    {% assign url = entity.fieldMedia.entity.image.url %}
+                    {% if entity.derivedFields.absoluteUrl %}
+                    {% assign url = entity.derivedFields.absoluteUrl %}
+                    {% endif %}
+                    <a class="file-download-with-icon" target="_blank"
+                      href="{{url}}" download="{{url}}"><i
+                        class="fas fa-download vads-u-padding-right--1"></i>Download
+                      {{entity.fieldMedia.entity.entityBundle | fileType}}</a>
+                    {% endif %}
+
+                  </div>
+                </div>
+              </div>
+              {% endif %}
+              {% endfor %}
+            </div>
+            {% endif %}
+
+          </div>
+
+          {% if title == 'Benefit Materials' %}
+          <div class="va-pagination">
+            <span class="max-control">
+              <a id="first-click">First</a>
+            </span>
+            <span class="va-pagination-prev">
+              <a id="pager-previous-click"
```

Comment:
These `a` elements will probably have to be converted to `button`s. They aren't changing to a new page, so they won't have an `href` and that will cause accessibility problems. There's a `va-button-link` class you can use to style a button like a link.

Reply:
:+1: @jbalboni